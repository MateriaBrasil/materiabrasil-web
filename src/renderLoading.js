import React from 'react'

import Loading from './Loading'

export default () => (
  <Loading style={{ padding: '40px', textAlign: 'center', width: '100%' }} />
)
