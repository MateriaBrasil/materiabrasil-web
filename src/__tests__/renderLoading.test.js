import renderer from 'react-test-renderer'
import renderLoading from '../renderLoading'

it('renders correctly', () => {
  const tree = renderer.create(renderLoading()).toJSON()
  expect(tree).toMatchSnapshot()
})
