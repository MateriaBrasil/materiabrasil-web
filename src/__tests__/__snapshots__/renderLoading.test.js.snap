// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "padding": "40px",
      "textAlign": "center",
      "width": "100%",
    }
  }
>
  <div>
    CircularProgress
  </div>
</div>
`;
